test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Name;
+
+
+public class UtilsTest {
+
+	@Test
+    public void nullInput_returnsTrue() {
+		String nullString = null;
+		boolean result = Utils.isAnyNull(nullString);
+        assertTrue(result);
+    }
+	
+	@Test
+    public void containsNull_returnsTrue() {
+		boolean result = Utils.isAnyNull("1", null, "3");
+        assertTrue(result);
+    }
```

Comment:
@Test
 public void containsNull_returnsTrue() {
	boolean result = Utils.isAnyNull("1", null, "3");
        assertTrue(result);
 }

Alignments should be like this.